editor/import/resource_importer_scene.cpp

```diff
@@ -1146,6 +1004,103 @@ void ResourceImporterScene::get_import_options(List<ImportOption> *r_options, in
 	}
 }
 
+String ResourceImporterScene::_import_hint_string(ResourceImporterScene::NodeHints p_hint) {
+	switch (p_hint) {
+		case NODE_HINT_NONE:
+			return "";
+		case NODE_HINT_NO_IMPORT:
+			return "noimp";
+		case NODE_HINT_COLLISION:
+			return "col";
+		case NODE_HINT_CONVEX_COLLISION:
+			return "convcol";
+		case NODE_HINT_COLLISION_ONLY:
+			return "colonly";
+		case NODE_HINT_CONVEX_COLLISION_ONLY:
+			return "convcolonly";
+		case NODE_HINT_SHAPE:
+			return "shape";
+		case NODE_HINT_CONVEX_SHAPE:
+			return "convshape";
+		case NODE_HINT_SHAPE_ONLY:
+			return "shapeonly";
+		case NODE_HINT_CONVEX_SHAPE_ONLY:
+			return "convshapeonly";
+		case NODE_HINT_RIGID:
+			return "rigid";
+		case NODE_HINT_NAVMESH:
+			return "navmesh";
+		case NODE_HINT_VEHICLE:
+			return "vehicle";
+		case NODE_HINT_WHEEL:
+			return "wheel";
+		default:
+			ERR_FAIL_V_MSG(String(), "Node import hint out of range");
+	}
+}
+
+String ResourceImporterScene::_import_hint_string(ResourceImporterScene::MaterialHints p_hint) {
+	switch (p_hint) {
+		case MATERIAL_HINT_ALPHA:
+			return "alpha";
+		case MATERIAL_HINT_VERTEX_COLOR:
+			return "vcol";
+		default:
+			ERR_FAIL_V_MSG(String(), "Material import hint out of range");
+	}
+}
+
+Pair<String, ResourceImporterScene::NodeHints> ResourceImporterScene::_get_node_import_hint(String p_name) {
```

Comment:
Is Pair from Godot? Godot tends to use structs for this.

Reply:
If it's not a godot data type, anything else in godot will need to import C++ std library etc.

Reply:
It's from core/templates/pair.h

Reply:
Yes :)

Reply:
> Godot tends to use structs for this.

Should I replace it with a struct? I also think that struct will look nicer.